main.go

```diff
@@ -190,13 +190,20 @@ func choose(modules []Module, pageSize int) []Module {
 	return updates
 }
 
-func update(modules []Module) {
+func update(modules []Module, hook string) {
 	for _, x := range modules {
 		fmt.Fprintf(color.Output, "Updating %s to version %s...\n", formatName(x, len(x.name)), formatTo(x))
 		out, err := exec.Command("go", "get", x.name).CombinedOutput()
 		if err != nil {
 			fmt.Printf("Error while updating %s: %s\n", x.name, string(out))
 		}
+		if hook != "" {
+			out, err := exec.Command(hook, x.name, x.from.String(), x.to.String()).CombinedOutput()
+			if err != nil {
+				fmt.Printf("Error while executing hook %s: %s\n", hook, string(out))
```

Comment:
I would prefer to `fatal` here instead of continuing to update the remaining modules. E.g. for the commit scenario, if the hook doesn't execute cleanly the next update might commit two updates. The same kind of issue would of course affect other scenarios.

Reply:
Indeed, this is better to fail fast here.
done in e5a5b83ebbe3c8f921c306cde42e316244caa273